.changeset/dirty-seahorses-look.md

```diff
@@ -0,0 +1,5 @@
+---
+"geohub": minor
+---
+
+feat: registering of algorithms
```

Comment:
merge two changesets into a changeset.
also, I think this can be a patch change since it is just for internal stac management tool.